props/saber_fett263_buttons.h

```diff
@@ -293,248 +660,3355 @@ class SaberFett263Buttons : public virtual PropBase {
+    }
+    current_preset_.Save();
+    UpdateFont(current_preset_.preset_num);
+  }
+
+  void SaveStyleSetting() {
+    char style_arg[10];
+    itoa(calc_, style_arg, 10);
+    current_preset_.SetStyle(blade_num_,style_parser.SetArgument(current_preset_.GetStyle(blade_num_), set_num_ + 2, style_arg));
+    current_preset_.Save();
+    // Stop location/size previews in Style Settings Mode
+    switch (set_num_) {
+      case LOCKUP_POSITION_ARG:
+        show_lockup_position_.Stop(blade_num_);
+        break;
+      case DRAG_SIZE_ARG:
+        show_drag_size_.Stop(blade_num_);
+        break;
+      case EMITTER_SIZE_ARG:
+        show_emitter_size_.Stop(blade_num_);
+      default:
+        break;
+    }
+    UpdateStyle(current_preset_.preset_num);
+    menu_type_ = MENU_STYLE_SETTING_SUB;
+    MenuSave();
+  }
+
+  void CancelStyleSetting() {
+    // Stop location/size preview in Style Settings Mode
+    switch (set_num_) {
+      case LOCKUP_POSITION_ARG:
+        show_lockup_position_.Stop(blade_num_);
+        break;
+      case DRAG_SIZE_ARG:
+        show_drag_size_.Stop(blade_num_);
+        break;
+      case EMITTER_SIZE_ARG:
+        show_emitter_size_.Stop(blade_num_);
+        break;
+      default:
+        break;
+    }
+    UpdateStyle(current_preset_.preset_num);
+    menu_type_ = MENU_STYLE_SETTING_SUB;
+    MenuCancel();
+  }
+#endif
+
+  // Saves New Color from Edit Mode Preview Styles to Preset
+  void NewColor(int blade, int effect) {
+    char new_color[32];
+    Color16 color_source;
+     switch (color_mode_) {
+      case COLOR_LIST:
+      case CC_COLOR_LIST:
+        color_source = Color16(color_list_[dial_]);
+        break;
+      default:
+        color_source = Color16(hsl_);
+        break;
+    }
+    itoa(Color16(color_source).r, new_color, 10);
+    strcat(new_color, ",");
+    itoa(Color16(color_source).g, new_color + strlen(new_color), 10);
+    strcat(new_color, ",");
+    itoa(Color16(color_source).b, new_color + strlen(new_color), 10);
+    current_preset_.SetStyle(blade,style_parser.SetArgument(current_preset_.GetStyle(blade), effect + 2, new_color));
+    if (color_mode_ != CC_COLOR_LIST) color_mode_ = NONE;
+  }
+
+  void Setup() override {
+    RestoreGestureState();
+  }
+
+#ifdef FETT263_SAVE_CHOREOGRAPHY
+  bool chdir(const char* dir) override {
+    bool ret = PropBase::chdir(dir);
+    RestoreChoreo();
+    clash_count_ = 0;
+    return ret;
+  }
+#endif
+
+  SoundQueue<16> sound_queue_;
+
+  enum SayType {
+    SAY_BATTERY,
+    SAY_DECIMAL,
+    SAY_WHOLE,
+  };
+
+  void TensValue(int number) {
+    if (number <= 20) {
+      sound_queue_.Play(SoundToPlay(&SFX_mnum, number - 1));
+    } else {
+      int tens = ((int)floorf(number / 10)) % 10;
+      number -= (tens * 10);
+      switch (tens) {
+        default:
+        case 2:
+          sound_queue_.Play(SoundToPlay(&SFX_mnum, (tens * 10) - 1));
+          break;
+        case 3:
+          sound_queue_.Play(SoundToPlay("thirty.wav"));
```

Comment:
It should be a simple search-and-replace

Reply:
Noted in separate comment but I do the editing on github.com, there isn't a search-and-replace I can find.  I can try to use find and replace in text editor but I believe that would cause github to say all lines are new (I had done this once before and it messed up the review process so I don't want to repeat).  So with how I'm doing the edits it would be manual or it would be done externally and then the entire prop would be new.

Reply:
Add a comment in the code as a reminder. I'll do the search-and-replace when it's checked in.

Reply:
comment added